packages/components/src/Select.tsx

```diff
@@ -9,6 +9,7 @@ import RSelect, {
   SingleValue as RSelectSingleValue,
 } from 'react-select';
 import RAsyncCreatableSelect, { AsyncCreatableProps as RAsyncCreatableProps } from 'react-select/async-creatable';
+import RCreatableSelect from 'react-select/creatable';
```

Comment:
We don't need the "async" behavior because we aren't populating the choices from an Ajax call (unlikes the "Tags" usage).